src/utils/LinkedFileStorage.ts

```diff
@@ -40,3 +49,16 @@ export abstract class LinkedFileStorage {
     return this.defaultStore.saveFile(filePath, fileContent);
   }
 }
+
+/**
+ *  get the asset include with cdn
```

Comment:
small change here. Its not just for CDN, could be anthing. 
So let's say:
"Get the full path of an asset based on the way LinkedFileStorage is configured"

Reply:
I will change this